crates/rspack_loader_swc/src/collect_ts_info.rs

```diff
@@ -39,10 +40,12 @@ pub fn collect_typescript_info(
             .map(|(id, v)| {
               let value = match v {
                 EnumMemberValue::Number(n) => Some(EvaluatedInlinableValue::new_number(n)),
-                EnumMemberValue::String(s) => Some(EvaluatedInlinableValue::new_string(s)),
+                EnumMemberValue::String(s) => Some(EvaluatedInlinableValue::new_string(
+                  s.to_atom_lossy().into_owned(),
```

Comment:
Maybe we should use `Wtf8Atom` here? Enum values may contain unpaired surrogates.